source/release-notes/1.7.txt

```diff
@@ -24,10 +41,6 @@ Patch Releases
 
 Issues Fixed:
 
-- Fixed bug introduced in v1.5.0 where mongosync could miss change
```

Comment:
I want Tim's input on whether to remove this. In my draft I modified the point instead of removing it https://docs.google.com/document/d/1vnHf-5qWZNXx_BtmB_3056O_8wxS5x2yz73C5Nfmasw/edit

Reply:
I'm okay just removing it. ORR is off by default so it's not particularly useful info anyway.